cfme/services/service_catalogs/ssui.py

```diff
@@ -35,32 +38,63 @@ def is_displayed(self):
 class OrderForm(ServiceCatalogsView):
```

Comment:
**REQUIRED**
What's the point to double the code? You can just import this view from cfme/services/service_catalogs/ui.py

Reply:
Cannot use same code as both have different views.

Reply:
@sshveta I don't see the difference. Please point me. `OrderForm` in ssui.py is the same as `OrderForm` in ui.py.

Reply:
@quarckster : form is same but the servicecatalog view is different for both of them 
OrderForm(ServiceCatalogsView):

Reply:
**REQUIRED**
In order to avoid code duplication, you may create a base `OrderForm` view and place it in cfme/services/service_catalogs/__init__.py. Then you can inherit it.

Reply:
Tried to add a common view for both UI's.
Using one orderForm now.